2.cellprofiler_processing/run_cellprofiler.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# convert jupyter notebook to a .py file once all variable changes are made
+jupyter nbconvert --to python example_cp.ipynb
+
+# run notebook through Python
+python example_cp.py
```

Comment:
Right now it looks like this will only rename one plate - I think this is ok, just making sure you're aware.

Reply:
Well... now that I think about it, I could make it into one whole function.

Reply:
Nevermind, it won't work because the .sqlite file is updated after every image, so this won't work. The user will have to rename the file after running the CellProfiler pipeline.